website/blog/2025-07-04-manage-ferretdb-databases-dbeaver.md

```diff
@@ -0,0 +1,97 @@
+
+## Why use DBeaver with FerretDB?
+
+DBeaver provides robust, built-in support for MongoDB.
+Because FerretDB is designed to be a truly open-source alternative to MongoDB, DBeaver can connect to your FerretDB instances as well.
+This powerful compatibility offers several compelling advantages:
+
+- **Familiarity for MongoDB users:** If you're already accustomed to DBeaver for MongoDB or other databases, you can immediately apply your knowledge to manage FerretDB instances.
+- **Visual data management:** Explore your PostgreSQL-backed document data through DBeaver's intuitive graphical interface, making it easier to understand, query, and debug.
+- **Simplified operations:** Perform common database operations visually, from Browse collections and documents to executing queries and basic administration, without extensive command-line interactions.
+- **Cross-database management:** Manage your FerretDB instances alongside any other SQL or NoSQL databases you use, all from a single DBeaver application.
+
+## Connecting DBeaver to FerretDB
+
+Connecting DBeaver to your FerretDB instance is a straightforward process, leveraging DBeaver's native MongoDB driver.
+Here's a step-by-step guide:
+
+1. **Ensure FerretDB is running:** Make sure your FerretDB instance is active and accessible.
+   If you haven't set it up yet, refer to our [FerretDB Installation Guide](https://docs.ferretdb.io/installation/ferretdb/).
+2. **Launch DBeaver:** Ensure that you have DBeaver installed on your system.
+   If you don't have it yet, download it from the [DBeaver website](https://dbeaver.io/download/).
+   Open the DBeaver application on your system.
+3. **Create a new database connection:** Click on "New Database Connection" from the toolbar or go to Database > New Database Connection.
+   In the "Connect to a database" wizard, type "MongoDB" in the search bar or navigate to the "NoSQL" section and select "MongoDB".
+   Click Next.
+4. **Configure connection settings:**
+   In the "Main" tab, fill in the connection details by entering the host, port, and authentication information for your FerretDB instance.
+   If you have [authentication enabled on your FerretDB instance](https://docs.ferretdb.io/security/authentication/), select the appropriate authentication method (e.g., "SCRAM-SHA-256") and enter your Username and Password.
+
+   ![A screenshot of DBeaver's 'New Database Connection' wizard](/img/blog/dbeaver-connection.png)
+
+5. **Test Connection:** Click the "Test Connection..." button.
+   If the connection is successful, you'll see a confirmation message.
+   Then click Finish.
+   The new connection will appear in your "Database Navigator."
+6. **Explore Your Data:** Expand your new FerretDB connection in the "Database Navigator." You can now browse databases, collections, view documents, and execute queries in the SQL editor or the dedicated MongoDB shell.
```

Comment:
Punctuation location is weird. Also is it really called `SQL editor`, I assumed it would be like script editor or like?

```suggestion
6. **Explore Your Data:** Expand your new FerretDB connection in the "Database Navigator". You can now browse databases, collections, view documents, and execute queries in the SQL editor or the dedicated MongoDB shell.
```